taskw/task.py

```diff
@@ -204,11 +204,12 @@ def set(self, key, value):
         """ Set a key's value regardless of whether a change is seen."""
         return self.__setitem__(key, value, force=True)
 
-    def serialized(self):
+    def serialized(self, include_immutable=True):
```

Comment:
This isn't technically directly related to these changes, but shows a slight difference in approach in my first implementation of this.  It's useful, though, in general for a side project and is backward-compatible, so I hope you'll forgive me for leaving this in.